src/components/layerselector/partials/layerselector.html

```diff
@@ -0,0 +1,5 @@
+<div ng-repeat="layer in backgroundLayers">
+  <div ng-click="onClick($event)" ng-show="!isClosed || $first" class="bgLayerDefault {{'bgLayer' + $index}} {{getClass(layer)}}">
```

Comment:
you could `ng-click="onClick(layer.id)" it would simplify the code here:
`scope.onClick = function(layerID) {`
Additionnaly I would use a button instead of a div for a clickable element.